test/library/standard/Random/performance/getNextPerf.chpl

```diff
@@ -0,0 +1,26 @@
+use Random, Time;
+
+config const perf = false;
+config const trials = if perf then 50_000_000 else 100;
+
+proc test(parallel, param parSafe) {
+  var rndStream = new RandomStream(uint(8), seed=314159265, parSafe=parSafe);
+  var t: Timer; t.start();
+  var a: uint;
+  if parallel then
+    forall 1..trials with (+reduce a) do
```

Comment:
Should we avoid using a reduction to avoid conflating reduction performance issues with getNext performance issues?

Reply:
I wondered about that too, but that should be negligible compared to the runtime of the loop itself (multiple seconds)